exposed-jdbc/src/main/kotlin/org/jetbrains/exposed/sql/statements/jdbc/JdbcDatabaseMetadataImpl.kt

```diff
@@ -172,6 +172,7 @@ class JdbcDatabaseMetadataImpl(database: String, val metadata: DatabaseMetaData)
         is SQLServerDialect -> defaultValue.trim('(', ')', '\'')
         is OracleDialect -> defaultValue.trim().trim('\'')
         is MysqlDialect -> defaultValue.substringAfter("b'").trim('\'').trim()
+        is PostgreSQLDialect -> defaultValue.trim()
```

Comment:
Looks like trim here and in default branch are wrong. With it it's impossible to set default value with whitespaces (like "  ").
Could you please remove the trim and adjust your test by replacing empty string with whitespaces?

Reply:
Thanks for pointing this out, I'll add another test to reproduce and will fix this.